akka-stream/src/main/scala/akka/stream/impl/StreamLayout.scala

```diff
@@ -89,10 +186,10 @@ private[akka] object StreamLayout {
         attributes)
     }
 
-    def grow(that: Module): Module = grow(that, Keep.left)
+    def compose(that: Module): Module = compose(that, Keep.left)
 
-    def grow[A, B, C](that: Module, f: (A, B) ⇒ C): Module = {
-      if (Debug) validate()
+    def compose[A, B, C](that: Module, f: (A, B) ⇒ C): Module = {
```

Comment:
ok. will hopefully have time during this evening

-- 
Cheers,
√
On 13 Jul 2015 17:24, "Konrad Malawski" <notifications@github.com> wrote:

> In akka-stream/src/main/scala/akka/stream/impl/StreamLayout.scala
> <https://github.com/akka/akka/pull/17921#discussion_r34474484>:
>
> >
> > -    def grow[A, B, C](that: Module, f: (A, B) ⇒ C): Module = {
> > -      if (Debug) validate()
> > +    def compose[A, B, C](that: Module, f: (A, B) ⇒ C): Module = {
>
> Thanks @viktorklang <https://github.com/viktorklang> :) Once fixed give
> it some OK commit message and we'll merge.
> The plan is to release 1.0 tomorrow.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/akka/akka/pull/17921/files#r34474484>.
>